modules/builds-configmaps-use-case-setting-command-line-arguments.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+//* builds/builds-configmaps.adoc
+
+[id="builds-configmaps-use-case-setting-command-line-arguments_{context}"]
+= Setting command-line arguments with ConfigMaps
+
+A ConfigMap can also be used to set the value of the command or arguments in a container. This is accomplished using the Kubernetes substitution syntax `$(VAR_NAME)`. Consider the following ConfigMaps:
+
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: special-config
+  namespace: default
+data:
+  special.how: very
+  special.type: charm
+----
+
+.Procedure
+
+* To inject values into the command line, you must consume the keys you want to use as environment variables, as in the Consuming in Environment Variables use case. Then you can refer to them in a container's command using the `$(VAR_NAME)` syntax.
```

Comment:
Also here you could refer to the pod to be more explicit. For example:

* To inject values into a command in a container...

Reply:
On further review, I am not sure where the 'Consuming in Environment Variables' use case is. Should this refer to the 'Consuming ConfigMaps in environment variables' use case?

Reply:
Also, should the requirement to consume the keys be a prerequisite here?